src/navier_stokes/INSVCStaggeredHierarchyIntegrator.cpp

```diff
@@ -1563,6 +1556,8 @@ void
 INSVCStaggeredHierarchyIntegrator::initializeCompositeHierarchyDataSpecialized(const double /*init_data_time*/,
                                                                                const bool initial_time)
 {
+    if (initial_time && d_Q_fcn) return;
```

Comment:
Why do you skip the projection in this case? I think the projection should work with div u = Q != 0.

Reply:
Yes, it can work in theory. However, what we learnt the hard way is that `Q` comes from from an underlying process that generally involves derivatives of quantities, which are not known at `t= 0`. More concretely, imagine we are doing phase change or a chemical reaction. The volume change `div u`  that needs to captured in `Q` would be some function of gradient of temperature or species concentration. These are not known at initial time. They gradually evolve during the simulation, which builds up `Q` over time. The best thing to do at `t = 0`, is basically not do anything!

Reply:
Sounds reasonable --- can you document this behavior somewhere in the class header, so that it shows up in the Doxygen documentation?